docs/wasmer-js/client/examples/handling-input-output/index.js

```diff
@@ -4,7 +4,7 @@ import { WasmFs } from '@wasmer/wasmfs';
 import { lowerI64Imports } from "@wasmer/wasm-transformer";
 
 // The file path to the wasi module we want to run
-const wasmFilePath = './static/duktape.wasm';
+const wasmFilePath = './quickjs.wasm';
```

Comment:
Is the path no longer in static?

Reply:
I think `.wasm` files need to be in `/static` otherwise `parcel` can't serve them...

Reply:
@syrusakbary @ChrisWhealy 

It's still in the `static/` directory, but this was a typo on my end. If you look at the other examples: 

https://github.com/wasmerio/docs.wasmer.io/blob/master/docs/wasmer-js/client/examples/transforming-wasi-modules/index.js#L7

The example files are in `static/`, but they are served from the root of the website. 

Thus the bug was, I never needed `static/` in the actual asset path 🙃